velox/common/file/FileSystems.h

```diff
@@ -43,6 +43,8 @@ struct FileOptions {
 
   std::unordered_map<std::string, std::string> values;
   memory::MemoryPool* pool{nullptr};
+  std::optional<int64_t> fileSize{
```

Comment:
- no need to explicitly initialize to std::nullopt
- comment goes on a line above the property; use /// for public APIs

```
/// If specified then can be trusted to be the file size.
std::optional<int64_t> fileSize;
```

Reply:
It seems the compiler disagrees: https://github.com/facebookincubator/velox/actions/runs/8636921920/job/23677942499?pr=9313#step:7:3240

Reply:
nvm I didn't see the follow up commit

Reply:
You're right, it was failing. Raised a fix here https://github.com/facebookincubator/velox/pull/9441.